### DISCUSSION TITLE
selective features

### DISCUSSION THREAD

**Discussion by MMYusuf** [pycaret/pycaret#3250] [Q&A]
Hi, 

Is there a way to apply preprocessing steps to a subset of the features? e.g. transform only 2 features out of 4 ? And still use all features for modelling 

Thanks!

**Comment by shreyabhadwal** [comment]
**ignore_features: list of string, default = None**
`ignore_features` can be used to ignore features during model training. It takes a list of strings with column names that are to be ignored.

Check it out here:
https://pycaret.gitbook.io/docs/get-started/preprocessing/data-preparation

Is this what you want to achieve?

**Reply by MMYusuf** [reply]
Thanks! I should make it a bit clearer indeed. I want to use all features but preprocess only a set of them.

**Reply by shreyabhadwal** [reply]
Ah... I am not sure if a Pycaret function is available for that?

A work around that I can think of is you slice your dataframe using loc/iloc, apply the setup function to the sliced dataframe and before training your model, merge those two dataframes.
